modules/cnv-uploading-local-disk-image-dv.adoc

```diff
@@ -3,10 +3,10 @@
 // * cnv/cnv_virtual_machines/cnv_virtual_disks/cnv-uploading-local-disk-images-block.adoc
 
 [id="cnv-uploading-local-disk-image-dv_{context}"]
-= Uploading a local disk image to a new DataVolume
+= Uploading a local disk image to a DataVolume
 
-You can use the `virtctl` CLI utility to upload a virtual machine disk image from
-a client machine to a DataVolume (DV) in your cluster. After you upload the image, 
+You can use the `virtctl` CLI utility to upload a local disk image from
+a client machine to a DataVolume (DV) in your cluster. After you upload the image,
 you can add it to a virtual machine.
 
```

Comment:
I think it would be useful to add some additional context about how this procedure can be used for uploading into existing and new DVs. The previous procedure in the assembly is about manually creating a DV, but the focus of this procedure is creating a new DV as part of the procedure. The procedure has asides that address how it can be modified if the DV exists but some (brief) additional context here would be helpful. 
(I know there's a line at the start of the assembly but by the time I've gone and created a DV I've forgotten about that)

Reply:
Good catch, I agree.